lib/api/shared/src/main/scala/net/wiringbits/api/ApiClient.scala

```diff
@@ -11,19 +11,28 @@ import scala.util.{Failure, Success, Try}
 
 trait ApiClient {
   def createUser(request: CreateUser.Request): Future[CreateUser.Response]
+
   def login(request: Login.Request): Future[Login.Response]
 
   def verifyEmail(request: VerifyEmail.Request): Future[VerifyEmail.Response]
+
   def forgotPassword(request: ForgotPassword.Request): Future[ForgotPassword.Response]
+
   def resetPassword(request: ResetPassword.Request): Future[ResetPassword.Response]
 
   def currentUser(jwt: String): Future[GetCurrentUser.Response]
+
   def updateUser(jwt: String, request: UpdateUser.Request): Future[UpdateUser.Response]
+
   def updatePassword(jwt: String, request: UpdatePassword.Request): Future[UpdatePassword.Response]
+
   def getUserLogs(jwt: String): Future[GetUserLogs.Response]
 
   def adminGetUserLogs(userId: UUID): Future[AdminGetUserLogs.Response]
+
   def adminGetUsers(): Future[AdminGetUsers.Response]
+
+  def getConfig(): Future[Config.Response]
```

Comment:
I'm not very sure about the name. Using `getConfig`/`Config.Response` doesn't sound very specific. I'll suggest you change it to make it clear that we're talking about getting frontend config `getFrontendConfig`, `getWebConfig`, `getWebAppConfig` or something else that sounds more explicit

Reply:
I'd suggest `def getEnvironmentConfig: Future[GetEnvironmentConfig.Response]`